zebra/zebra_srv6_vty.c

```diff
@@ -448,6 +448,59 @@ static void do_show_srv6_sid_detail(struct vty *vty, json_object **json,
 	}
 }
 
+static void do_show_srv6_sid(struct vty *vty, json_object **json,
```

Comment:
`do_show_srv6_sid` and `do_show_srv6_sid_detail` has a lot of duplicate code? can you just refactor the same function to do both?

Reply:
@Jafaral Many thanks for the review. I will refactor the code and update the PR.

Reply:
Done